runtime/oti/VMHelpers.hpp

```diff
@@ -1381,7 +1381,7 @@ class VM_VMHelpers
 			}
 			break;
 		case J9NtcBoolean:
-			*returnStorage = (UDATA)(U_8)*returnStorage;
+			*returnStorage = (UDATA)(U_8)(*returnStorage & 1);
```

Comment:
It's clearer if the `& 1` is bubbled out of the read/cast brackets: `((UDATA)(U_8)*returnStorage) & 1;`